README.md

```diff
@@ -1,26 +1,20 @@
 Zendesk Unity3D Plugin
-==========================
-
-This is a Unity plugin that wraps the iOS and Android Zendesk SDKs. It also allows optional inclusion of the Urban Airship Unity SDK to assist with push notification features.
-
-## Third Party Packages
+======================
 
+This is a Unity plugin that wraps the iOS and Android Zendesk SDKs.
 
 ## Requirements
 
 - OSX or Linux operating system (The plugin is not supported on Windows)
 - Unity 5.0+
-- Xcode 7.0+ (if building for iOS)
-- Android SDK with the latest support repo and libraries installed. (Required to build the plugin, even if you are just using iOS)
-- Note that you must have the "Android Support Library" installed. You can download this using the Android SDK Manager. 
```

Comment:
I can't remember why this was required, but I remember a couple of months ago, I couldn't build until I did it. Can you remember what the deal was here, and why it's no longer needed?